openbb_terminal/cryptocurrency/onchain/onchain_controller.py

```diff
@@ -427,6 +430,50 @@ def call_whales(self, other_args: List[str]):
                 else None,
             )
 
+    @log_start_end(log=logger)
+    def call_topledger(self, other_args: List[str]):
+        """Process topledger command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="topledger",
+            description="""
+                    Display on-chain data from topledger.
+                    [Source: https://docs.topledger.xyz/]
```

Comment:
This documentation page is horrible. There are literally no examples of the code, end points, data returned, or even explanations.